configs/debian8_clang/0.3.0/BUILD

```diff
@@ -24,3 +24,28 @@ java_runtime(
     srcs = [],
     java_home = "/usr/lib/jvm/java-8-openjdk-amd64",
 )
+
+# Latest RBE Debian8 container.
+# Update every time when a new container is released.
+alias(
+    name = "rbe_debian8",
+    actual = ":rbe_debian8_r328903",
+)
+
+# RBE Debian8 r328903
+platform(
+    name = "rbe_debian8_r328903",
+    constraint_values = [
+        "@bazel_tools//platforms:x86_64",
+        "@bazel_tools//platforms:linux",
+        "@bazel_tools//tools/cpp:clang",
+        "//constraints:support_msan",
```

Comment:
just to double check, resolution is such that the toolchain's exec_compatible_with can be a subset of the values here (i.e., the non msan toolchain also satisfies this platform)?